docs/en/UI/AspNetCore/Tag-Helpers/Modals.md

```diff
@@ -0,0 +1,89 @@
+# Modals
+
+## Introduction
+
+`abp-modal` is a main element to create a modal.
+
+Basic usage:
+
+````xml
+<abp-button button-type="Primary" data-toggle="modal" data-target="#myModal">Launch modal</abp-button>
+
+<abp-modal centered="true" size="Large" id="myModal">
+   <abp-modal-header title="Modal title"></abp-modal-header>
+   <abp-modal-body>
+       Woohoo, you're reading this text in a modal!
+   </abp-modal-body>
+   <abp-modal-footer buttons="Close"></abp-modal-footer>
+</abp-modal>
+````
+
+
+
+## Demo
+
+See the [modals demo page](https://bootstrap-taghelpers.abp.io/Components/Modals) to see it in action.
+
+## Attributes
+
+### centered
+
+A value indicates the positioning of the modal. Should be one of the following values:
+
+* `false` (default value)
+* `true`
+
+Example:
+
+````xml
+<abp-alert alert-type="Warning">
```

Comment:
This `abp-alert` example shouldn't be here.

Reply:
Removed the wrong sample totally.